data-prepper-plugins/mongodb/src/main/java/org/opensearch/dataprepper/plugins/mongo/export/ExportPartitionWorker.java

```diff
@@ -164,13 +162,15 @@ public void run() {
                     }
 
                     try {
-                        final String record = cursor.next().toJson(writerSettings);
+                        final String record = cursor.next().toJson(JSON_WRITER_SETTINGS);
                         final long bytes = record.getBytes().length;
                         bytesReceivedSummary.record(bytes);
                         // The version number is the export time minus some overlap to ensure new stream events still get priority
                         final long eventVersionNumber = (exportStartTime - VERSION_OVERLAP_TIME_FOR_EXPORT.toMillis()) * 1_000;
                         final Event event = recordConverter.convert(record, exportStartTime, eventVersionNumber);
-
+                        // event.put(DEFAULT_ID_MAPPING_FIELD_NAME, event.get(DOCUMENTDB_ID_FIELD_NAME, Object.class));
```

Comment:
This will be configurable in follow up PR. If customer enables the configuration, we will write to customer provided field. Same for stream processing.